source/_integrations/leviosa_shades.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: Leviosa Motor Blinds Zone hub
+description: Instructions on how to integrate the Leviosa Zone hub into Home Assistant.
+ha_category:
+  - Cover
+ha_iot_class: Local Push
+ha_release: 2021.4
+ha_domain: leviosa_shades
+ha_codeowners:
+  - '@altersis'
+ha_config_flow: true
+ha_platforms:
+  - cover
+---
+
+This integration allows you to control [Leviosa Zone hubs](https://leviosashades.com/products/leviosa-zone) from [Leviosa Motor Shades](https://leviosashades.com/).
+
+{% include integrations/config_flow.md %}
+
+## Things to keep in mind
+
+- Each Leviosa Zone manages up to 6 groups of shades.
+- If you have more than one Leviosa Zone in your Network, Home Assistant will display a list, with one IP address for each Leviosa Zone discovered, in the same order as they were discovered. Once a Leviosa Zone is configured in Home Assistant, it will not appear in this list. Refer to the next section to find the IP address of each zone group.
+- If you have only one Leviosa Zone, then you'll be taken directly to the screen to enter the Leviosa Zone name and shade group names.
+- You can name the shade groups as you wish. They do not have to match the names already in the Leviosa App.
+- A `cover` entity for each shade group will be ready for you. An additional `cover` entity is created with the name of the zone (prefixed `all_`) to move all groups in a Leviosa Zone at the same time.
+
+## Getting Zone hub IP Addresses from the app
+
+When you have multiple Leviosa Zones, you'll need to know which IP address is associated with each Leviosa Zone hub. The easiest way to do this is to:
+
+- Open the Leviosa App.
+- Press and hold on any shade group name. After 1 second, a new screen will appear.
+- Select the ‘Name’ tab at the top.
+- The screen will show the IP address of the Leviosa Zone that shade group is on.
+
+## Services to move your blinds to intermediate positions
+
+When you want your blinds to go to intermediate positions defined by you, set these intermediate positions with the [Leviosa remote controller](https://cdn.shopify.com/s/files/1/1346/0347/files/Leviosa_Shades_Programming_instructions.pdf?). These intermediate positions are stored in each Leviosa shade motor, and two services in Home Assistant tell shades in each group to move there. If you do not program intermediate points, the shade will fully travel to the upper or lower limit when these services are used. Once the intermediate positions are set, you can:
+
+- Use `leviosa_shades.next_up_pos` to move up to the next intermediate position.
+- Use `leviosa_shades.next_down_pos` to move down to the next intermediate position.
+
+Both services need the `entity_id` of the shade group that you want to move
```

Comment:
That seems weird? This should be an entity service, which can accept a device or area as well. This comment makes me think it is not implemented correctly.

Reply:
Hi. Thanks again for taking a look at this. I'll try to explain in more detail: The service is in fact an entity service. In the Leviosa Zone one channel typically represents all the shades you have in one room.  Imagine that we have a living room with five shades, and all of them go up or down at the same time. All five shades would be set by the installer under channel one of the Leviosa Zone; that channel is represented in HASS using one entity. When configuring the Leviosa Zone through the UI, the user would enter the name 'Family room' for channel one. Calling either service with 'cover.family_room' will make all shades move to the next up/down position (these positions are programmed into the motor, so HASS can only tell the channel to go up/down to the next  position programmed.  This is why, at least in my eyes there is no need to be able to pass an area to the service. Makes sense?

Reply:
Sure but if it uses an entity service, this statement is untrue, thus should be reworded or removed.

Reply:
You say that because it appears to imply that HASS groups would not work? if that is the case I know they would work and would gladly change the wording to indicate that you can pass a HASS group or one or more cover entities. Just want to make sure that I address the point of your comment.

Reply:
No I say that because this line:

> Both services need the `entity_id` of the shade group that you want to move

Is simply not true when using entity services. An area or device can be provided alternatively to those service calls as part of their target. This line should be removed.

Reply:
Thanks for your patience. I think I understand where the problem is, and agree to remove the line. I can now see that the nomenclature used by the product (Leviosa Zone, which controls one or more Groups of Shades) can create some confusion with the Hass concepts of Zone and Group. Will correct the document in a moment.